src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -0,0 +1,436 @@
+
+### The `onscreenschange` event
+
+The only thing missing now is a way to detect when my screen setup changes. A new event,
+`onscreenschange` (note: plural "screens"), does exactly that: it fires whenever the screen constellation
+is modified. This can happen when the resolution of one of the connected screens changes or when a
+new or an existing screen is (physically or virtually in the case of Sidecar) plugged in or unplugged.
+
+Note that you need to look up the new screen details asynchronously, the `screenschange` event
+itself does not provide this data. This may [change in the
+future](https://github.com/webscreens/window-placement/issues/28). For now you
+can look up the screen details by calling `window.getScreens()` as shown below.
+
+```js
+window.addEventListener('screenschange', async (e) => {
+  console.log('I am there, but mostly useless', e);
+  const details = await window.getScreens();
+});
+```
+
+### New fullscreen options
+
+Until now, you could request that elements be displayed in fullscreen mode via the aptly named
+[`requestFullScreen()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/requestFullscreen)
+method. The method takes an `options` parameter where you can pass
+[`FullscreenOptions`](https://developer.mozilla.org/en-US/docs/Web/API/FullscreenOptions). So far, its only property has been
+[`navigationUI`](https://developer.mozilla.org/en-US/docs/Web/API/FullscreenOptions/navigationUI).
+The Multi-Screen Window Placement API adds a new `screen` property that allows you to determine
+which screen to start the fullscreen view on. For example, if you want to make the primary screen fullscreen:
+
+```js
+try {
+  const primaryScreen = (await getScreens()).filter((screen) => screen.primary)[0];
+  await document.body.requestFullscreen({ screen: primaryScreen });
+} catch (err) {
+  console.error(err.name, err.message);
+}
+```
+
+### Polyfill
+
+It is not possible to polyfill the Multi-Screen Window Placement API, but you
+can shim its shape so you can code exclusively against the new API:
+
+```js
+if (!("getScreens" in window)) {
+  // Returning a one-element array with the current screen,
+  // noting that there might be more.
+  window.getScreens = async () => [window.screen];
+  // Returning `false`, noting that this might be a lie.
+  window.isMultiScreen = async () => false;
+}
+```
+
+The other aspects of the API—the `onscreenschange` event and the `screen` property of the
+`FullscreenOptions`—would simply never fire or silently be ignored respectively by non-supporting
+browsers.
+
+## Demo
+
+If you are anything like me, you keep a close eye on the development of the various cryptocurrencies.
+(In reality I very much do not, but, for the sake of this article, just assume I do.) To
+keep track of the cryptocurrencies that I own, I have developed a web app that allows me to watch
+the markets in all life situations, like from the comfort of my bed, where I have a decent
```

Comment:
```suggestion
the markets in all life situations, such as from the comfort of my bed, where I have a decent
```

Reply:
I know this is correcting my own requested change. Please forgive me. I was tired when I did this.